shared/wallets/send-form/index.js

```diff
@@ -1,54 +1,76 @@
 // @flow
 import * as React from 'react'
-import * as Kb from '../../common-adapters'
-import * as Styles from '../../styles'
-import Body from './body/container'
-import Header from './header'
+import SendFormRoot from './root-container'
+import LinkExisting from '../link-existing/container'
+import CreateNewAccount from '../create-account/container'
 
 type Props = {|
   isRequest: boolean,
-  bannerInfo?: string,
-  isProcessing?: boolean,
   onClose: () => void,
 |}
 
-const SendForm = (props: Props) => (
-  <Kb.MaybePopup onClose={props.onClose}>
-    <Kb.Box2 direction="vertical" style={styles.container}>
-      <Header />
-      <Body bannerInfo={props.bannerInfo} isProcessing={props.isProcessing} isRequest={props.isRequest} />
-    </Kb.Box2>
-    <Kb.Box2 direction="horizontal" centerChildren={true} fullWidth={true} style={styles.textContainer}>
-      <Kb.Text type="BodySmallSemibold" style={styles.textColor}>
-        Powered by{' '}
-        <Kb.Text
-          type="BodySmallSemiboldSecondaryLink"
-          isLink={true}
-          onClickURL="https://stellar.org"
-          style={styles.textColor}
-        >
-          stellar
-        </Kb.Text>
-      </Kb.Text>
-    </Kb.Box2>
-  </Kb.MaybePopup>
-)
+type State = {
+  currentScreen: 'root' | 'link-existing' | 'create-new-account',
+}
 
-const styles = Styles.styleSheetCreate({
-  container: Styles.platformStyles({
-    isElectron: {
-      height: 525,
-      width: 360,
-    },
-  }),
-  textContainer: {
-    position: 'absolute',
-    textAlign: 'center',
-    bottom: -26, // TODO: tweak this number, maybe make it calculated from the text's line height and a global margin
-  },
-  textColor: {
-    color: Styles.globalColors.white_40,
-  },
-})
+class SendForm extends React.PureComponent<Props, State> {
+  state = {
+    currentScreen: 'root',
+  }
+  linkExisting = () => {
+    this.setState(() => ({
+      currentScreen: 'link-existing',
+    }))
+  }
+  createNewAccount = () => {
+    this.setState(() => ({
+      currentScreen: 'create-new-account',
+    }))
+  }
+  backToRoot = () => {
+    this.setState(() => ({
+      currentScreen: 'root',
+    }))
+  }
+  render() {
+    switch (this.state.currentScreen) {
+      case 'root':
+        return (
+          <SendFormRoot
+            isRequest={this.props.isRequest}
+            onClose={this.props.onClose}
+            onLinkAccount={this.linkExisting}
+            onCreateNewAccount={this.createNewAccount}
```

Comment:
After creating a new account it gets selected in the UI behind the send form, but the account details never load on closing the form. It should load the details, but I think it'd also be fine to not select the newly created account when created from the send form

Reply:
Hmm sorry I'm not sure I understand correctly. The new account is selected because of [this action](https://github.com/keybase/client/pull/13993/files#diff-314fc39c85ff1807227a38a2848d7851R283) which is dispatched when [created or linked](https://github.com/keybase/client/blob/9d4c6568298b6ba71fd473f404b7bd7d6c18143c/shared/actions/wallets.js#L417) happens, which also triggers `loadAccounts` [here](https://github.com/keybase/client/blob/9d4c6568298b6ba71fd473f404b7bd7d6c18143c/shared/actions/wallets.js#L381). So it seems to me if the form is closed early, we wouldn't select. If we select, it must have been a successful creation or linking, which would also definitely trigger loading account details.

Reply:
Yeah, that all makes sense. I just mean when it's selected in the background, the assets / tx history in the background perma-spinner even after closing the dialog:
<img width="761" alt="dev_keybase" src="https://user-images.githubusercontent.com/11968340/46625097-a1cff880-cb00-11e8-8ecd-c28f8062d12a.png">

Just checked and this happens on master too, we can leave it for another ticket if you want.